search/search_repository.go

```diff
@@ -275,9 +275,18 @@ func parseMap(queryMap map[string]interface{}, q *Query) {
 			q.Name = key
 			s := string(concreteVal)
 			q.Value = &s
+			if q.Name == "iteration" || q.Name == "area" {
+				if strings.HasSuffix(s, "/**") {
```

Comment:
I am not getting the last of part. When this is not going to work?

Reply:
This PR is only considering ID column to lookup for child iteration/area. Since IDs are UUIDs, there won't be any `/**` character anywhere in the value.

Reply:
I did get that and that is why I'm so concerned.

But then you would have to learn a new syntax when you want to allow searching for an iteration/area that is given by any other field. I don't see the real value in this other than a quick fix with problems down the line. I mean the `/**` doesn't even work for paths of an iteration where it seems to make the most sense (`/foo/bar/**`) because nobody is holding you back from naming your iteration `**`.

Reply:
If path & name requires similar syntax, we can do it in a separate PR.

Reply:
That is exactly my point, path, resolved_path, name, number, start date, ... they probably require **different** suffixes because of the domain and allowed vocabulary. And as a user I probably should not care by what field I query (e.g. name, path, id, number) the syntax should remain the same on query level.

Reply:
So, what should I do with this PR? Avoid using `/**` altogether and use `"child":true` attribute?
Please suggest.